resources/lib/context_animatedart.py

```diff
@@ -5,27 +5,45 @@
     Contextmenu for Animated art
 '''
 
+import os, sys
 import xbmc
 import xbmcgui
 from metadatautils import MetadataUtils
-from utils import log_msg
+if sys.version_info.major == 3:
+    from resources.lib.utils import log_msg
+else:
+    from utils import log_msg
+    
 
 # pylint: disable-msg=invalid-constant-name
 
 
 def get_imdb_id(win, metadatautils):
     '''get imdbnumber for listitem'''
     content_type = win.getProperty("contenttype")
-    imdb_id = xbmc.getInfoLabel("ListItem.IMDBNumber").decode('utf-8')
+    if sys.version_info.major == 3:
+        imdb_id = xbmc.getInfoLabel("ListItem.IMDBNumber")
+    else:
+        imdb_id = xbmc.getInfoLabel("ListItem.IMDBNumber").decode('utf-8')
```

Comment:
Since you need this if/else block so much, you should put it in a function in utils module and then import it.

For instance:
```python
def decode_utf8(item):
    if sys.version_info.major == 3:
        return item
    else:
        return item.decode('utf-8')
```

Then you only need one line to do this decoding - e.g.
```python
imdb_id = decode_utf8(xbmc.getInfoLabel("ListItem.IMDBNumber"))
```

Reply:
valid point. The code actually had functions for that already, but wasn't used much. I've replaced the if checks now.